modules/covid_vaccine/app/jobs/covid_vaccine/expanded_scheduled_submission_job.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+
+module CovidVaccine
+  class ExpandedScheduledSubmissionJob
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options retry: false
+
+    def perform
+      Rails.logger.info('Covid_Vaccine Expanded_Scheduled_Submission: Start')
+
+      count = CovidVaccine::V0::ExpandedRegistrationSubmission.where(state: 'enrollment_pending').count
+      Rails.logger.info('Covid_Vaccine Expanded_Scheduled_Submission Start: enrollment_pending count', count: count)
+
+    # Sorting by DESC here because currently if a record fails MPI lookup, it remains in state=enrollment_pending,
+    # so starting at the beginning we may never see the new records as only the MPI error records would process due to limit
+    # with anticipated volume and execution every 15 minutes we should be able to process all new records 
+    # todo: implement task to find and deal with MPI error records and enrollment_out_of_bounds records
+      CovidVaccine::V0::ExpandedRegistrationSubmission.where(state: 'enrollment_pending').order('created_at DESC').limit(1000).map do |submission|
+        CovidVaccine::ExpandedSubmissionJob.perform_async(submission.id)
+      end
+
+      count = CovidVaccine::V0::ExpandedRegistrationSubmission.where(state: 'enrollment_pending').count
+      Rails.logger.info('Covid_Vaccine Expanded_Scheduled_Submission Complete: enrollment_pending count', count: count)
+
+    #   the subtask being called does not raise errors, but keeping this in case some other issue causes this job to fail
+    rescue => e
+        handle_errors(e)
+    end
+
+    def handle_errors(ex)
+      Rails.logger.error('Covid_Vaccine Expanded_Scheduled_Submission: Failed')
```

Comment:
Confirming that when raised, this will include the record `id` in the log? Would be very helpful for debugging.

Reply:
@batemapf I added logging with record_id  to [expanded_submission_job](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/covid_vaccine/app/jobs/covid_vaccine/expanded_submission_job.rb), which this Job invokes for each record.  we need to capture the individual record processing errors there.

Reply:
The logging actually happens in the service class when the error occurs, so this is handled already.